### DISCUSSION TITLE
Welcome to Legal-Text-Analytics Discussions!

### DISCUSSION THREAD

**Discussion by bwaltl** [Liquid-Legal-Institute/Legal-Text-Analytics#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alddagas** [comment]
I am a legal designer working in the field of legal design data science (which does not exist yet : ).I have popped here occasionally and collected enough courage to knock on the door if someone is at home.

**Reply by bwaltl** [reply]
Hi, great to know that somebody is around interested in getting in touch. We have a strong footprint in Legal Design and Data Science, however, both dots aren't sufficiently connected until now.

**Reply by Alddagas** [reply]
Nice :).Here in Finland, this issue has not been a hot topic, at least in the way I would prefer it to be viewed. The language area lacks proper language models and suffers the same norm tweaking as other EU members.
